src/app/containers/Caption/index.jsx

```diff
@@ -29,18 +47,35 @@ const renderCaption = (paragraphBlocks, imageCaptionOffscreenText, script) => (
   </Caption>
 );
 
-const CaptionContainer = ({ block }) => {
-  const { script, imageCaptionOffscreenText } = useContext(ServiceContext);
+const CaptionContainer = ({ block, type }) => {
+  const {
+    script,
+    imageCaptionOffscreenText,
+    videoCaptionOffscreenText,
+  } = useContext(ServiceContext);
+  const offscreenText = chooseOffscreenText(
+    type,
+    videoCaptionOffscreenText,
+    imageCaptionOffscreenText,
+    script === 'arabic'
```

Comment:
This doesn't scale very well. The `useContext(ServiceContext)` above gives access to the service config, you can read `defaultCaptionOffscreenText` out of it :)